doc/getting-started/apl-nuc.rst

```diff
@@ -236,23 +237,15 @@ partition. Follow these steps:
 
    .. code-block:: none
 
-      # clr-boot-manager set-timeout 20
-      # clr-boot-manager update
+      $ sudo clr-boot-manager set-timeout 20
+      $ sudo clr-boot-manager update
 
-#. Add new user
+#. Enable and start the ssh server, so you can login remotely using an ssh client:
```

Comment:
If I remember correctly, this is not needed and the `sshd` service is enabled by default. To be more precise, the `sshd.socket` is active (not the `sshd` service).

Reply:
not a key point, we can mark it as the "optional" to make sure ssh is available from remotely access,
ssh is a check point here, even without "ssh enabled" steps

Reply:
Please remove this. It's not needed and it enabling `sshd` a different it already is. It can help us simplify this GSG.
```suggestion
```